docs/meetings.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Weekly Community Meetings
+---
+
+# Weekly Meetings
+----------------
+
+## Engineering meeting
+ - Monday at 11:00AM - 11:30AM PT
```

Comment:
Add link to calendar event similar to community meeting here as well.

Reply:
I dont have a public link for now. We'll have to move forward with this for now